src/sonic-config-engine/tests/t0-sample-graph.xml

```diff
@@ -260,6 +260,20 @@
           <MacAddress i:nil="true"/>
           <SecondarySubnets/>
         </VlanInterface>
+        <VlanInterface>
+          <Name>Vlan98</Name>
+          <AttachTo>fortyGigE0/100;PortChannel01;PortChannel03</AttachTo>
```

Comment:
why this fortyGige0/100 port does not appear in the  DATAACLEGRESS ports?

Reply:
It's `Ethernet100`